### PR TITLE
Add solutions to Project Euler problem 56, 58, 63

### PR DESCRIPTION
**Pull request by nehamsoni**

### CODE REVIEW

**Comment by poyea** [comment]
Thanks. Only 3 files added? I'll review it later.

File: project_euler/problem_56/sol1.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from __future__ import print_function
```
Comment (poyea) [inline project_euler/problem_56/sol1.py]: We don't need this anymore. As stated in our contribution guide, Python means Python 3.x+.

File: project_euler/problem_56/sol1.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+try:
+	xrange		#Python 2
+except NameError:
+	xrange = range	#Python 3
```
Comment (poyea) [inline project_euler/problem_56/sol1.py]: Just use `range()`, for the same reason.

File: project_euler/problem_56/sol1.py (hunk 1 above)
Comment (cclauss) [reply project_euler/problem_56/sol1.py]: https://docs.python.org/3/howto/pyporting.html#prevent-compatibility-regressions

This directive changes how Python 2 operates.  It makes it a syntax error in Python 2 to add a legacy __print__ statement.  This prevents Python 2 developers from breaking Python 3 compatibility.

Comment (poyea) [reply project_euler/problem_56/sol1.py]: Right. If we consider any forward compatibility, this should be here. It doesn't matter a lot as long as it's on the first line.

**Pull request closed without merging**